### ISSUE TITLE
A method to handle interest

### ISSUE THREAD

**Issue by didiermiller** [firefly-iii/firefly-iii#6512]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

On my (savings-)bank account, which is part of my mortgage, I get a monthly deposit of interest. The importer tool does not pick this up (for reasons I do not understand fully), but I assume that the issue is because there is no source account.

### Solution

A solution would be to calculate the interest rate each month based on the current value of a select able bank account. This would be a hefty change I assume as it now clearly says "this option does nothing" and is purely cosmetic.

Basically it boils down to:
`(Bank account total / interest percentage) / interval`
Where interval is 1 for 1 year, or 12 for per month, etc.

### What are alternatives?

An alternative would be to add an option to the automatic transactions to allow a certain percentage (+ select which bankaccount) instead of a fixed value. 

Or, if you select a savings bank account, add the percentage interest there in the option screen and calculate the interest (this can be a checkmark setting) there.

**Comment by JC5** [comment]
The reason interest is not integrated into the application is because there are many ways to calculate and payout interest. Banks have different rates depending on the account type and how much money is in the account. Plus, interest rates tend to change over time and this affects the calculations as well.

At the end of the day, it's easier for the user to simply use whatever the bank paid out, instead of my trying to copy all of their invisible logic into Firefly III (and still getting it wrong).

So this will not happen I'm afraid.
